runtime.md

```diff
@@ -99,7 +97,7 @@ This operation MUST return the state of a container as specified in the [State](
 `create <container-id> <path-to-bundle>`
 
 This operation MUST [generate an error](#errors) if it is not provided a path to the bundle and the container ID to associate with the container.
-If the ID provided is not unique across all containers within the scope of the runtime, or is not valid in any other way, the implementation MUST [generate an error](#errors) and a new container MUST NOT be created.
+If the ID provided is not unique across all containers within the [scope of the container](#scope-of-a-container), or is not valid in any other way, the implementation MUST [generate an error](#errors) and a new container MUST NOT be created.
```

Comment:
> > I think the "Scope of a Container" section is really about the scope of this state-sharing
>
> I don't quite see that, maybe the scope of a container section needs to be clarified.

My previous attempt at clarifying the connection is [here][1].  I'm happy to cherry-pick that into this PR and make any necessary adjustments if that helps.  But I don't see how the runtime is expected to enforce ID uniqueness without using something like its state sharing mechanism.

[1]: https://github.com/duglin/runtime-spec/pull/1/commits/2fff5454fb7f20beec6110ff8725fc8507f4cce8